nixos/lib/make-options-doc/default.nix

```diff
@@ -153,16 +157,20 @@ in rec {
   # Convert options.json into an XML file.
   # The actual generation of the xml file is done in nix purely for the convenience
   # of not having to generate the xml some other way
-  optionsXML = pkgs.runCommand "options.xml" {} ''
+  optionsXML = pkgs.runCommand "options.xml" {
+    nativeBuildInputs = [ pkgs.nix.out ];
+  } ''
     export NIX_STORE_DIR=$TMPDIR/store
     export NIX_STATE_DIR=$TMPDIR/state
-    ${pkgs.nix}/bin/nix-instantiate \
+    nix-instantiate \
       --eval --xml --strict ${./optionsJSONtoXML.nix} \
       --argstr file ${optionsJSON}/share/doc/nixos/options.json \
       > "$out"
   '';
 
-  optionsDocBook = pkgs.runCommand "options-docbook.xml" {} ''
+  optionsDocBook = pkgs.runCommand "options-docbook.xml" {
+    nativeBuildInputs = [ pkgs.libxslt.bin pkgs.libxslt.bin pkgs.python3Minimal ];
```

Comment:
libxslt included twice, not really a problem but unnecessary.

Reply:
Was included in original commit, forgot to clean it up (even after noticing it myself), oops. Will fix asap, thanks for pointing it out.

Reply:
```suggestion
    nativeBuildInputs = [ pkgs.libxslt.bin pkgs.python3Minimal ];
```

Reply:
Why are you suggesting this after someone already mentioned it? As I've said many times before, please read the discussion on a PR before commenting, it just makes you make duplicate comments.